build.gradle.kts

```
@@ -214,51 +253,52 @@ tasks.processResources {
     }
 }
 
-java {
-    withSourcesJar()
-}
-
 tasks.create<Jar>("javadocJar") {
     from(tasks.javadoc)
     archiveClassifier.set("javadoc")
 }
 
+tasks.named<ProcessResources>("processGametestResources") {
+    duplicatesStrategy = DuplicatesStrategy.WARN
+}
+
 tasks.jar {
     from("LICENSE")
     manifest {
-        attributes(mapOf(
+        attributes(
             "Implementation-Title"     to modName,
             "Implementation-Version"   to project.version,
             "Implementation-Vendor"    to "Team Galacticraft",
             "Implementation-Timestamp" to DateTimeFormatter.ISO_DATE_TIME,
             "Maven-Artifact"           to "$modGroup:$modName:$project.version"
```

```suggestion
            "Maven-Artifact"           to "$modGroup:$modName:${project.version}"
```
missing curly brackets for more than one word

---

Fixed in f7fd2d664718bf7ee5f8d0c19c7f964e71fbad8d